anta/tests/connectivity.py

```diff
@@ -66,11 +66,11 @@ def render(self, template: AntaTemplate) -> list[AntaCommand]:
     @AntaTest.anta_test
     def test(self) -> None:
         """Main test function for VerifyReachability."""
-        self.result.is_success()
-
         for command, host in zip(self.instance_commands, self.inputs.hosts):
+            host_result = self.result.add(description=str(host), inputs=host)
```

Comment:
should we find a way to signal a test returns atomic test result for documentation puproses?